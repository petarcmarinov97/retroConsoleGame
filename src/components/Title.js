import React, { Component } from 'react';
import { Typography } from 'antd';


export default class Title extends Component {
    render() {
        const { Title } = Typography;
        const { gameTitle } = this.props;

        return (
            <Title level={4}>{gameTitle}</Title>
        )
    }
}
