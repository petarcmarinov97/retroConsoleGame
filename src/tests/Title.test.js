import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from "enzyme-to-json";
import Title from '../components/Title';

describe('Title tests', () => {

    let wrapper;
    let gameTitle = "Tetris";

    beforeEach(() => {
        wrapper = shallow(
            <Title gameTitle={gameTitle} />);
    })

    describe('Snapshots', () => {
        it("should render correctly", () => {
            expect(shallowToJson(wrapper)).toMatchSnapshot();
        })
    })
})