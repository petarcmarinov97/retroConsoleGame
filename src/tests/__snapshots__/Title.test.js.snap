// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title tests Snapshots should render correctly 1`] = `
<Title
  level={4}
>
  Tetris
</Title>
`;
